src/components/payment/PaymentSection.js

```diff
@@ -154,11 +158,18 @@ class PaymentSection extends React.Component {
             ''
           )}
           {priceGoesUpOn && nextDiscountPrice ? (
-            <P>
-              Ticket price goes up to{' '}
-              {formatPrice(currency, nextDiscountPrice, DEFAULT_VAT_RATE)} on{' '}
-              {priceGoesUpOn}
-            </P>
+            <React.Fragment>
+              <P>
+                Ticket price goes up to{' '}
+                {formatPrice(currency, nextDiscountPrice, DEFAULT_VAT_RATE)} on{' '}
+                {priceGoesUpOn}
+              </P>
+              {showCountDownTimer && (
```

Comment:
@ric9176 unless Paul disagrees, I think we should always display the countdown if the price is discounted (meaning priceGoesUpOn is not null)

Reply:
Yes, we are inside the conditional render for ```priceGoesUpOn && nextDiscountPrice``` which should be ok. I'll also add some logic for displaying the counter tho so that it won't show if the date is less than now, bc priceGoesUpOn may not get updated to a falsy value (as it's currently manual) so we'd end up with a counter saying you have zero time.